src/view/writer.js

```diff
@@ -887,6 +887,8 @@ function wrapChildren( parent, startOffset, endOffset, attribute ) {
 	let offsetChange = 0;
 
 	for ( let position of wrapPositions ) {
+		position.offset -= offsetChange;
```

Comment:
It is explained by commit message.

This got removed, probably due to mistake and was causing a bug. This exact line of code was there long time ago in repo history.